docs/test-failures.md

```diff
@@ -42,3 +42,14 @@ upcoming changes, and maintainers of other projects can subscribe.
 
 Without strict versioning, this contract between projects and upstream
 will always be brittle - so regular communication is a must.
+
+# Troubleshooting
+## Create Cluster Fails
+There is an issue in provisioning the cluster.
+
+Test failure will look like:
+```error during make -C /workspace/kubernetes-anywhere WAIT_FOR_KUBECONFIG=y deploy: exit status 2```
```

Comment:
When previewing, the two lines are actually in one single line. Please check it. (Adding an empty line between them might fix this.)